lib/rubocop/cop/style/method_call_parentheses.rb

```diff
@@ -50,6 +46,13 @@ def same_name_assignment?(node)
               return false if asgn_node.send_type?
             end
 
+            if asgn_node.masgn_type?
+              mlhs_node, _mrhs_node = *asgn_node
+              vars = mlhs_node.children.map(&:source)
```

Comment:
@bbatsov Mapping to `children` instead of `source` now. Not sure how good this idea is compared to mapping to `source`. 

Actually, I did not get what you meant by `symbol identifiers`, hence came up with this.
If mapping is still a bad approach, can you point me to some existing code which uses `symbol identifiers`?